src/test/java/redis/clients/jedis/tests/commands/MigrateTest.java

```diff
@@ -154,6 +154,16 @@ public void migrateAuth() {
     assertNull(jedis.get(bfoo));
   }
 
+  @Test
+  public void migrateAuth2() {
+    jedis.set("foo", "bar");
+    destAuth.aclSetUser("acljedis", "on", "allcommands", "allkeys", ">fizzbuzz");
+    assertEquals("OK",
+            jedis.migrate(host, portAuth, dbAuth, timeout, new MigrateParams().auth2("acljedis", "fizzbuzz"), "foo"));
```

Comment:
You can consider migrating data from `dest`/`destAuth` to `jedis`. You won't need a new user in that case.

You can also a binary test. You may use one of the `dest`s for string test and another for binary test.